src/Cpp/1-getting-started/2-HelloFroge/Main.cpp

```diff
@@ -331,11 +306,8 @@ ID3D11VertexShader* LoadVertexShader(const std::wstring_view filePath, ID3D10Blo
     }
 
     ID3D11VertexShader* vertexShader;
-    if (FAILED(g_Device->CreateVertexShader(
-        (*vertexShaderBlob)->GetBufferPointer(),
-        (*vertexShaderBlob)->GetBufferSize(),
-        nullptr,
-        &vertexShader)))
+    if (FAILED(g_Device->CreateVertexShader((*vertexShaderBlob)->GetBufferPointer(), (*vertexShaderBlob)->GetBufferSize(), nullptr,
```

Comment:
again that >1 param then line break thingy

Reply:
```
    if (FAILED(g_Device->CreateVertexShader(
            (*vertexShaderBlob)->GetBufferPointer(), (*vertexShaderBlob)->GetBufferSize(), nullptr, &vertexShader)))
    {
```
is this better?

Reply:
It now looks like this:
```cpp
    if (FAILED(g_Device->CreateVertexShader(
            (*vertexShaderBlob)->GetBufferPointer(),
            (*vertexShaderBlob)->GetBufferSize(),
            nullptr,
            &vertexShader)))
    {
```

Reply:
That doesnt look half bad, is it possible to indent with 4 spaces instead of 8? _beingpicky_

Reply:
There's a few different options I can select from:
- BAS_Align (in configuration: Align) Align parameters on the open bracket, e.g.:
```cpp
someLongFunction(argument1,
                 argument2);
```

- BAS_DontAlign (in configuration: DontAlign) Don’t align, instead use ContinuationIndentWidth, e.g.:
```cpp
someLongFunction(argument1,
    argument2);
```

- BAS_AlwaysBreak (in configuration: AlwaysBreak) Always break after an open bracket, if the parameters don’t fit on a single line, e.g.:
```cpp
someLongFunction(
    argument1, argument2);
```

- BAS_BlockIndent (in configuration: BlockIndent) Always break after an open bracket, if the parameters don’t fit on a single line. Closing brackets will be placed on a new line. E.g.:
```cpp
someLongFunction(
    argument1, argument2
)
```

Reply:
BAS_DontAlign would be my preference

Reply:
And i might be able to live with odd indents, other than 4 in that instance. possibly an obscure undocumented paramter... editorconfig has a massive catalog as well

Reply:
I think it does 4 spaces relative to the function call, I have set `ContinuationIndent` to 4 but it doesn't seem to affect this.
```cpp
if (FAILED(...
12341234arg
```

Reply:
Is VS respecting that indentation at "all times"? then that should be OK for now

Reply:
I don't use VS, so I don't know how to check that.

Reply:
I will also have a look at clang-format with VS over the week I think

Reply:
I think i want `BAS_Align` that is the most reasonable out of the lot, after revisiting those options